app/src/androidTest/java/org/mozilla/fenix/ui/docs/assets.md

```diff
@@ -0,0 +1,33 @@
+# README
+_This document is intended to explain specifics in regards to test assets for the Espresso/UI Automator Android UI tests._
+
+# Test application
+* Remote test page managed by Mozilla Mobile Test Engineering at https://github.com/mozilla-mobile/testapp
+* File type download page: https://storage.googleapis.com/mobile_test_assets/test_app/downloads.htm
```

Comment:
Nit: missing 'l' at the end, otherwise the page does not load

Reply:
fixed